tools/roslaunch/src/roslaunch/xmlloader.py

```diff
@@ -180,9 +181,15 @@ def resolve_args(self, args, context):
         Wrapper around substitution_args.resolve_args to set common parameters
         """
         # resolve_args gets called a lot, so we optimize by testing for dollar sign before resolving
+       
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.